package.json

```diff
@@ -33,31 +33,31 @@
     "did-veres-one": "^12.1.1",
     "edv-client": "^4.0.1",
     "express-async-handler": "^1.1.4",
-    "jsonld-signatures": "^5.0.1",
+    "jsonld-signatures": "^5.1.0",
     "ocapld": "^2.0.0",
-    "webkms-client": "^2.1.0"
+    "webkms-client": "^2.3.0"
   },
   "peerDependencies": {
-    "bedrock": "^3.1.0",
-    "bedrock-account": "^3.0.0",
+    "bedrock": "^3.1.1",
+    "bedrock-account": "^4.0.0",
```

Comment:
As you pointed out, since this module does not `require('bedrock-account');` anywhere in the `lib` folder, bedrock-account is not a necessary peerDependency and should be removed.  The usage in the test suite is fine and should be upgraded.